api/turing/config/config.go

```diff
@@ -337,7 +340,10 @@ type OpenapiConfig struct {
 	// SpecFile specifies the file path containing OpenAPI v3 spec
 	SpecFile string
 	// Optional. Defines a configuration to be used for serving Swagger UI as a single-page app
+	// If this is provided, it will use the bundled yaml instead.
 	SwaggerUIConfig *SinglePageApplicationConfig
+	// Optional. Overrides the file before running the Swagger UI.
+	SwaggerUIConfigOverrideFile *string
```

Comment:
Shall it be named as `SpecOverrideFile` or something like lt? It's used not only by Swagger UI, but also for API validation

Reply:
ok done